src/lib/utils/__tests__/mergeSearchParameters-test.ts

```diff
@@ -367,4 +367,31 @@ describe('mergeSearchParameters', () => {
       })
     );
   });
+
+  it('merges and dedupes `ruleContexts` parameters', () => {
+    const actual = merge(
+      algoliasearchHelper.SearchParameters.make({
+        ruleContexts: ['ais-genre-comedy'],
```

Comment:
The `make` function is not yet properly typed in the JS helper at this point and is not aware of the `ruleContexts` search parameter. Rebasing should work.

Reply:
see https://github.com/algolia/instantsearch.js/pull/3940 which will have a fix for that